pkg/forklift-api/webhooks/validating-webhook/admitters/secret-admitter.go

```diff
@@ -72,6 +74,17 @@ func (admitter *SecretAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissio
 
 func (admitter *SecretAdmitter) validateProviderSecret() *admissionv1.AdmissionResponse {
 	if createdForProviderType, ok := admitter.secret.GetLabels()["createdForProviderType"]; ok {
+		if admitter.ar.Request.Operation == admissionv1.Update && createdForProviderType == api.Ova.String() {
+			log.Info("secret admitter for OVA")
+			return &admissionv1.AdmissionResponse{
+				Allowed: false,
+				Result: &metav1.Status{
+					Code:    http.StatusForbidden,
+					Message: "Updating URL field is not allowed, please create a new provider",
```

Comment:
do we know that it's specifically the URL that is being updated here?

Reply:
I see, so in this case, let's document it

Reply:
and also I don't think we should ask the user the create another provider, how about:
```suggestion
					Message: "Updating the URL field of an existing OVA provider is not allowed",
```